src/api/schemas/extendedQueryNode.graphql

```diff
@@ -22,7 +27,7 @@ type Channel {
 
   avatarPhotoUrl: String
 
-  owner: Member!
+  member: Membership!
```

Comment:
Don't know why, but in the gist, there is no `member` field in `Channel`, and also there are no `channels` on `Membership`. 
Not sure what we should do with that. If the schema is wrong I think you should also add `channels` to `Membership`

Reply:
it's a temporary solution/assumption, currently we are waiting for the query node guys to update that. @kdembler do we have more info about that?

Reply:
As far as I can tell this is currently blocked by some missing functionality in Hydra so we have to wait. Let's leave it out for now and update when the new query node gets deployed